scripts/install_addon.py

```diff
@@ -1,9 +1,27 @@
-import os
+import optparse, sys
 import bpy
 
-version = "/CR/" + os.environ["CR_VERSION"] + ".zip"
+args = sys.argv[sys.argv.index("--") + 1:]
 
-bpy.ops.preferences.addon_install(overwrite=True, filepath=version)
+
+# add arguments for installing and enabling the addon
+parser = optparse.OptionParser()
+# add option to install an addon from a file
+parser.add_option('-i', "--install", action="store_true", dest="install")
+parser.add_option("-e", "--enable", action="store_false", dest="install")
```

Comment:
nitpick:
The `-e` option does not really make sense, because if conflicts with the `-i` switch and `addon_enable` gets called regardless.
I would rather add `default=False` to the `-i` option.
```suggestion
parser.add_option('-i', "--install", action="store_true", dest="install", default=False)
```

Reply:
Agree, not sure where that was going, esp a year on. Removed the -e enable option